src/module-elasticsuite-catalog/view/adminhtml/web/js/form/element/product-sorter.js

```diff
@@ -155,6 +157,11 @@ define([
             return this.products().length < this.countTotalProducts();
         },
 
+        searchProducts: function () {
+            // this.currentSize(this.pageSize);
```

Comment:
Dead code :)

Reply:
@romainruaud If you only really mean unnecessary commented code (line 161), I agree.

Reply:
yep, I don't want to have unnecessary commented code

Reply:
It was a POC... :) I removed this.